app/uk/gov/hmrc/cgtpropertydisposals/connectors/SubscriptionConnector.scala

```diff
@@ -63,7 +67,7 @@ class SubscriptionConnectorImpl @Inject()(http: HttpClient, val config: Services
       http
         .post(subscribeUrl, Json.toJson(subscriptionRequest), headers)(
           implicitly[Writes[JsValue]],
-          hc.copy(authorization = None),
+          hc.copy(authorization = None), //FIXME: remove when DES APIs are dropped to IST0
```

Comment:
we'd keep this even when DES are in their test environment - we send the `Authorization` header separately with the DES bearer token

Reply:
will it need to be removed for PROD?

Reply:
no, this will work in prod as well - we don't need to send the "mdtp" bearer token to des when we make the call to them

Reply:
can we get rid of the `FIXME`'s here?

Reply:
I have removed them.